org.osgi.test.common/src/main/java/org/osgi/test/common/dictionary/Dictionaries.java

```diff
@@ -322,4 +323,26 @@ public V remove(Object key) {
 			return map.remove(key);
 		}
 	}
+
+	/**
+	 * Utility method for creating a {@link Dictionary} of service properties.
+	 *
+	 * @param arguments number of arguments must be divisible by 2 and odd
+	 *            arguments should be strings ({@link String#valueOf(Object)}
+	 *            will be used)
+	 * @return dictionary
+	 */
+	public static Dictionary<String, Object> of(Object... arguments) {
```

Comment:
Ick. I don't like this much at all. I would rather do something like https://osgi.org/gitweb/build.git/blob/refs/heads/core:/org.osgi.test.support/src/org/osgi/test/support/map/Maps.java which is a `Map.of` style thing for pre-Java 9.

Reply:
removed.